iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -65,7 +65,7 @@ class PortManager_test : public Test
 
     uint16_t m_instIdCounter, m_eventIdCounter, m_sIdCounter;
 
-    iox::ProcessName_t m_ProcessName{"TestProcess"};
+    iox::RuntimeName_t m_runtimeName{"TestProcess"};
```

Comment:
Could be ``TestRuntime``?

Reply:
I chose `TestApp`, to make show that they often share the same name.